ddtrace/appsec/_iast/_metrics.py

```diff
@@ -80,7 +65,7 @@ def _set_iast_error_metric(msg: Text) -> None:
         }
         telemetry.telemetry_writer.add_log(TELEMETRY_LOG_LEVEL.ERROR, msg, stack_trace=stack_trace, tags=tags)
     except Exception:
-        log.warning("Error reporting ASM logs metrics", exc_info=True)
+        log.warning("[IAST] Error reporting logs metrics", exc_info=True)
```

Comment:
Silly question... 😅  Generally speaking, is the scope hierarchy:

> AppSec > ASM > IAST ?

So this change is narrowing the scope of the warning log from ASM -> IAST?

Reply:
A long, long time ago, in a branch far, far away... xD when IAST was first starting out, it did follow that hierarchy. But for years now, within AppSec, ASM and IAST have been on the same level, that is, each operates independently, and you can enable one without using the other. Last year, we refactored the code but left an error like that behind, which might cause confusion, as IAST does not depend on ASM :+1: